website/content/docs/job-specification/identity.mdx

```diff
@@ -10,7 +10,9 @@ description: |-
 
 <Placement
   groups={[
+    ['job', 'group', 'service', 'identity'],
     ['job', 'group', 'task', 'identity'],
+    ['job', 'group', 'task', 'service', 'identity'],
   ]}
```

Comment:
I'm not sure about the ordering here. `job -> group -> task -> identity` is arguably  the most important placement, but moving it first make things look out of order.